### ISSUE TITLE
Make all fields in response consistent, e.g. in camelCase or underscore style, etc. 

### ISSUE THREAD

**Issue by chenzongxiong** [litentry/api-gateway#78]
Should we consider to make the fileds in response consistent across the whole `api-gateway` system?

**Comment by bdmason** [comment]
@chenzongxiong yes, with camelCase, in the index.ts file of the gateway we just need to move the underscores from the prefixes.

**Comment by geastwood** [comment]
Is this still relevant @chenzongxiong?

**Comment by chenzongxiong** [comment]
@geastwood this is feature good to have. I'm not sure whether anyone implements this feature or not. 

When I extended api-gateway with `discord`, `twitter`, `discourse`, etc, I found inconsitent naming style in the response. 
That's why I create this issue. And I think we need to decide it as soon as possible since `api-gateway` is still not used by `mcp`. 

@bdmason I found `rn-app` has its own Query in api-gateway. Before we decide which style to choose, we should also ask @franjs . Otherwise, we may break their dependencies.
